tests/common/test_image.py

```diff
@@ -91,7 +91,7 @@ def test_get_intrinsics_from_exif_focal_length(self, mock_init, mock_lookup):
             "FocalPlaneResolutionUnit": 2,
         }
         image = Image(np.random.randint(low=0, high=255, size=(im_h, im_w, 3)), exif_data)
-        computed_intrinsics = image.get_intrinsics_from_exif(default_focal_length_factor=DEFAULT_FOCAL_LENGTH_FACTOR)
+        computed_intrinsics = image.get_intrinsics(default_focal_length_factor=DEFAULT_FOCAL_LENGTH_FACTOR)
```

Comment:
it looks like exif_data is being passed to the image class, so arent these tests checking for it? do we need focal length factor then?

Reply:
This case tests the intrinsics are from exif data when it is available. It does not matter if the factor is passed or not. Is that what you are asking?

Reply:
yes, so why not just use the get_intrinsics_from_exif function without the default?

Reply:
and maybe add another unit test where we dont pass exif data and call get_intrinsics with a default focal length factor?

Reply:
There was already one at line20.
Anyway, I will re-arrange the test cases, making sure all the cases are covered.

Reply:
Hello @akshay-krishnan , I've refined the test cases a bit. Now the first five tests `get_intrinsics_from_exif()` method: `no exif`, `no tags` and reading focal length from three different tags.  The following two tests `get_intrinsics()` method: `no exif` and `with exif`.